### DISCUSSION TITLE
PUT et PATCH Anomalie Adresse semblent redondants

### DISCUSSION THREAD

**Discussion by alexisthethe** [before-interop/anomalieAdresse#14] [Polls]
(lié à la discussion https://github.com/before-interop/anomalieAdresse/discussions/11)

Pour mettre à jour une anomalie adresse, il y a 2 endpoints possibles :

PUT /{anomalieAdresseId}
PATCH /{anomalieAdresseId}
Cela me semble redondant et porte à confusion sur l'utilisation de l'API. Ne faudrait-il pas plutôt choisir la méthode qui sera utilisée plutôt que de laisser plusieurs possibilités ?
D'autant plus que j'ai l'impression que dans la réalité on aura besoin uniquement du PATCH (lorsqu'un OC va mettre un jour un ticket il va mettre à jour partiellement le ticket à chaque fois, il ne va jamais faire un PUT).

Qu'en pensez-vous ?
@ggrebert @aminvielledebat @acostaIFT @ChristopheDaudenthun @aminesalmen-rekik

**Comment by ggrebert** [comment]
Pas d'avis en particulier, par contre je ne suis vraiment pas pour utiliser uniquement PUT.

Soit les 2, soit uniquement PATCH.

**Comment by ggrebert** [comment]
décision du 6 septembre: on garde les 2
